tsconfig.json

```diff
@@ -2,15 +2,15 @@
   "compilerOptions": {
     "baseUrl": "src",
     "outDir": "build/dist",
-    "module": "commonjs",
+    "module": "esnext",
     "target": "es5",
     "lib": [
       "es5",
       "es6",
       "dom"
     ],
     "sourceMap": true,
-    "jsx": "react",
+    "jsx": "react-jsx",
```

Comment:
The following changes are being made to your tsconfig.json file:
  - compilerOptions.jsx must be react-jsx (to support the new JSX transform in React 17)